common/dynamicconfig/constants.go

```diff
@@ -1991,6 +1991,11 @@ archivalQueueProcessor`,
 		10,
 		`WorkflowExecutionMaxInFlightUpdates is the max number of updates that can be in-flight (admitted but not yet completed) for any given workflow execution. Set to zero to disable limit.`,
 	)
+	WorkflowExecutionMaxInFlightUpdatePayloads = NewNamespaceIntSetting(
+		"history.maxInFlightUpdatePayloads",
+		10*1024*1024,
```

Comment:
We have to start somewhere. Open for better suggestions!

Reply:
It is just a log warn. I think it is ok.